src/awkward/operations/ak_transform.py

```diff
@@ -425,6 +425,40 @@ def transform(
     `"none"`
         The output arrays will not be given parameters.
 
+
+    Performance Tipp
```

Comment:
```suggestion
    Performance Tip
```

Reply:
Thanks! Good to have feedback from a native speaker 👍